src/scripts/modules/media/body/body.coffee

```diff
@@ -92,13 +93,19 @@ define (require) ->
       @cc.handleClose = (eventData) ->
         @handleClosed(eventData, $body[0])
 
-      @cc.init(settings.conceptCoach.url)
-
       @cc.on('ui.close', @cc.handleClose)
       @cc.on('open', @cc.handleOpen)
-      @cc.on('book.update', @changePageByUuids)
+      @cc.on('book.update', @updatePageFromCCNav)
+
+      Backbone.on('window:resize', =>
+        @cc.handleResize()
+      )
+
+    updateCCOptions: ->
+      options = @getOptionsForCoach()
+      @cc.setOptions(options)
 
-    changePageByUuids: ({collectionUUID, moduleUUID, link}) =>
+    updatePageFromCCNav: ({collectionUUID, moduleUUID, link}) =>
```

Comment:
if you read the full diff, it's also changed here: https://github.com/Connexions/webview/pull/1288/files#diff-91497210f7b8c5503232d20830246bf9R98